.vuepress/config.js

```diff
@@ -51,6 +51,7 @@ module.exports = {
             '/learn/what_is_meilisearch/comparison_to_alternatives',
             '/learn/what_is_meilisearch/telemetry',
             '/learn/what_is_meilisearch/contact',
+            '/learn/what_is_meilisearch/language',
```

Comment:
Can we have  '/learn/what_is_meilisearch/contact', at the bottom instead?

Reply:
Sure, probably a good idea!